arm-dependencies.sls

```diff
@@ -0,0 +1,85 @@
+arm-dependencies:
+  pkg.installed:
+    - pkgs:
+      - g++-aarch64-linux-gnu
+      - g++-arm-linux-gnueabihf
+
+{% set path = 'https://servo-rust.s3.amazonaws.com/ARM/' %}
+{% set targets = [{ 'name': 'arm-linux-gnueabihf', 
+                    'symlink_name': 'arm-unknown-linux-gnueabihf',
+                    'version': 'v1',
+                    'local_name': 'armhf-trusty-libs.tgz',
+                    'hash': 'd9a31ed488e4f848efcd07f71aa167fc73252da2a2c3b53ba8216100e2b4302b5ccd273b27c434ad189650652a1877607d328ff6b8e1edb5aa68a8927c617b49',
+                   },
```

Comment:
style nit: I tend to like my close brackets lining up vertically with my open brackets (this one and the next one are one space too far over to the right).